### DISCUSSION TITLE
Bug Report: "source_xp/WebmHandler.hx:59: characters 15-16 : Expected }" error I can't fix

### DISCUSSION THREAD

**Discussion by zenolth** [KadeArchive/Kade-Engine#1042] [Q&A]
Hello, I was running lime test windows but this showed up "source_xp/WebmHandler.hx:59: characters 15-16 : Expected }" can someone tell me the fix?

**Comment by zenolth** [comment]
PS : I got error in Main.hx on line 4 too.

**Comment by MoezTrigui** [comment]
same i got that eoor too
